frontend/src/service/Api.service.ts

```diff
@@ -0,0 +1,88 @@
+import { AuthService } from './Auth.service';
+import { Err, ErrorService } from './Error.service';
+import { EnvService } from './Env.service';
+import { GraphQLService, IGraphQLError } from './GraphQL.service';
+import { Url } from '../entity/Url';
+
+interface IGraphQLSchemaQuery {
+  authQuery: IGraphQLSchemaAuthQuery;
+}
+
+interface IGraphQLSchemaAuthQuery {
+  URLs?: IGraphQLSchemaURL[];
```

Comment:
I think the `?` after `URL` is not needed.

Here is the API definition for AuthQuery: https://github.com/short-d/short/blob/44a1208e6f2e5b08ff42294d3f88e236f025cf42/backend/app/adapter/graphql/schema.go#L20

From the API definition, `URLs` is not nullable.

Reply:
@byliuyang , please do correct me if I am wrong.
Doesn't this depend on the fetch query we have? If I am querying only for URLs, is it expected to return other members like ChangeLog?

Reply:
@rohithbalaji123 You will only get URLs.